lib/logstash/outputs/s3/temporary_file_factory.rb

```diff
@@ -64,13 +65,8 @@ def current_time
         end
 
         def generate_name
-          filename = "ls.s3.#{SecureRandom.uuid}.#{current_time}"
-
-          if tags.size > 0
-            "#{filename}.tag_#{tags.join('.')}.part#{counter}.#{extension}"
-          else
-            "#{filename}.part#{counter}.#{extension}"
-          end
+          filename = "#{current_time}"
+          "#{filename}.#{extension}"
```

Comment:
Are we sure they don't rely on the filename anywhere else?

Reply:
Im pretty sure it's the only place, but i'll double check

Reply:
Nice (: